transformer_thermal_model/schemas/thermal_model/onaf_switch.py

```diff
@@ -50,3 +78,25 @@ def check_consistency(self) -> Self:
         if self.fans_status is None and self.temperature_threshold is None:
             raise ValueError("Either 'fans_status' or 'temperature_threshold' must be provided.")
         return self
+
+
+class ONAFSwitch(ONAFSwitchBase):
+    """Class representing the ONAF (Oil Natural Air Forced) cooling switch settings.
+
+    This class includes the ONAN parameters to be used when the transformer switches to ONAN cooling.
+    """
+
+    onan_parameters: ONANParameters = Field(
+        ..., description="ONAN parameters to be used when the transformer switches to ONAN cooling."
+    )
+
+
+class ThreeWindingONAFSwitch(ONAFSwitchBase):
```

Comment:
This name will become very long... so please consider if you have any other creative naming options 
```suggestion
class ThreeWindingCoolingSwitchSettings(CoolingSwitchBase):
```